File: leet/Java/untitled/src/DesignHashSet.java
public class DesignHashSet {
    Integer[] hashset = new Integer[10000];

    public int hash(int key) {
        return key % 10000;
    }

    public void add(int key) {
        if (this.contains(key))
            return
        int idx = this.hash(key);
        if (this.hashset[idx] == null)
            this.hashset[idx] = new int[]{key};
        else
            this.hashset[idx].append(key);
    }

    public void remove(int key) {
        if (! this.contains(key))
            return
        int idx = this.hash(key);
        if (this.hashset[idx] != null) {
            arr = this.hashset[idx];
            delete
        }
    }

    public boolean contains(int key) {
        int idx = this.hash(key);
        if (this.hashset[idx] == null)
            return null
        else {
            int arr = this.hashset[idx];
            for (int val: arr){
                if (val) {
                    return true;
                }
            return false;
            }
            return
        }
        return false;
    }

}

File: leet/Design_HashSet.py
class DesignHashSet(object):
    def __init__(self):
        self.hashset = [None] * 10000

    def hash(self, key):
        return key % 10000

    def add(self, key):
        if self.contains(key):
            return
        idx = self.hash(key)
        if self.hashset[idx] is None:
            self.hashset[idx] = [key]
        else:
            self.hashset[idx].append(key)

    def remove(self, key):
        if not self.contains(key):
            return
        idx = self.hash(key)
        if self.hashset[idx] is not None:
            arr = self.hashset[idx]
            del arr[arr.index(key)]

    def contains(self, key):
        idx = self.hash(key)
        if self.hashset[idx] is None:
            return None
        else:
            arr = self.hashset[idx]
            print(self.hashset)
            print(arr)
            return any(val == key for val in arr)

if __name__ == "__main__":
    hashSet = DesignHashSet()
    hashSet.add(1)
    hashSet.add(2)

    hashSet.contains(2)

    #print(hashSet.hashset)